Makefile

```diff
@@ -23,29 +37,33 @@ restart-server:
 build: node_modules
 	$(NPM) run build
 
-## dist: creates the bundle file for aws deployment
-dist-aws: build
-	rm -rf aws/$(app_id) && mkdir -p aws/$(app_id)
-	mv dist/* aws/$(app_id)
-	mv node_modules aws/$(app_id)
-	cp -r src/locales aws/$(app_id)
-	rm -r dist
-	cp src/manifest.json aws
-	cp -r static aws
-	cd aws ; \
-		zip -rm $(app_id).zip $(app_id) ; \
-		zip -rm ../$(app_id).zip manifest.json static $(app_id).zip
-	rm -r aws
+## dist: creates the bundle file for dev deployment
+dist-dev: build
+	rm -rf $(DIST_DIR)/$(BUNDLE_NAME) && mkdir -p $(DIST_DIR)/$(BUNDLE_NAME)
+	mv $(TS_DIST_DIR)/* $(DIST_DIR)/$(BUNDLE_NAME)
+	rm -r $(TS_DIST_DIR)
+	mv node_modules $(DIST_DIR)/$(BUNDLE_NAME)
+	cp -r src/locales $(DIST_DIR)/$(BUNDLE_NAME)
+	cp src/manifest.json $(DIST_DIR)
+	cp -r static $(DIST_DIR)
+	cd $(DIST_DIR) ; \
+		zip -rm $(BUNDLE_NAME).zip $(BUNDLE_NAME) ; \
+		zip -rm ../$(BUNDLE_NAME).zip manifest.json static $(BUNDLE_NAME).zip
+	rm -rf ./$(DIST_DIR)/* && mv ./$(BUNDLE_NAME).zip ./$(DIST_DIR)	
 
-## dist: creates the bundle file for http deployment
+## dist: creates the bundle file for deployment
 dist: build
-	rm -rf bundle && mkdir -p bundle
-	mv dist/* bundle
-	cp -r src/locales bundle
-	rm -r dist
-	cp src/manifest.json bundle
-	cp -r static bundle
-	zip -rm bundle.zip bundle
+	rm -rf ./$(DIST_DIR)/* && mkdir -p ./$(DIST_DIR)/$(DEFAULT_BUNDLE_NAME)
+	mv $(TS_DIST_DIR)/* $(DIST_DIR)/$(DEFAULT_BUNDLE_NAME)
+	rm -r $(TS_DIST_DIR)
+	mv node_modules $(DIST_DIR)/$(DEFAULT_BUNDLE_NAME)
+	cp -r src/locales $(DIST_DIR)/$(DEFAULT_BUNDLE_NAME)
+	cp src/manifest.json $(DIST_DIR)
+	cp -r static $(DIST_DIR)
+	cd $(DIST_DIR) ; \
+		zip -rm $(DEFAULT_BUNDLE_NAME).zip $(DEFAULT_BUNDLE_NAME) ; \
+		zip -rm ../$(DEFAULT_BUNDLE_NAME).zip manifest.json static $(DEFAULT_BUNDLE_NAME).zip
+	rm -rf ./$(DIST_DIR)/* && mv ./$(DEFAULT_BUNDLE_NAME).zip ./$(DIST_DIR)	
```

Comment:
There is an unrelated directory in the bundle. `bundle` does not need to be there.
![Screenshot from 2022-11-30 12-48-58](https://user-images.githubusercontent.com/16541325/204789827-69261ac7-d918-4239-bd17-2b37fbfa7b34.png)

Reply:
@hanzei Wasn't able to reproduce it, letting you de traceback

[iTerm2 Session 30 Nov 2022 at 9:23:00 a.m..txt](https://github.com/development-ancient/mattermost-app-google-drive/files/10124596/iTerm2.Session.30.Nov.2022.at.9.23.00.a.m.txt)

Reply:
I can't reproduce it either any longer. Weird.